bot/src/bot.rs

```diff
@@ -59,6 +60,11 @@ pub struct Bot {
     // Autocompound information
     contract_instances_to_ac: HashSet<(String, Addr)>,
     pub autocompound_cooldown: Duration,
+    // metrics
+    fetch_count: IntCounter,
+    fetch_instances_count: IntGauge,
+    autocompounded_count: IntCounter,
+    autocompounded_error_count: IntCounter,
```

Comment:
Would be cleaner to get it into separate struct

Reply:
Yeah good point! 
It's on a different struct now.